cnd/migrations.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+function navigate_to_last_folder() {
```

Comment:
```suggestion
function navigate_to_first_folder() {
```
This navigates to the first folder by lexical order. I find the naming confusing.

Reply:
It is not the first, it's the last (using `tail`).
Maybe it can be done in a better way.